server/src/main/java/org/apache/iotdb/db/qp/physical/PhysicalPlan.java

```diff
@@ -48,6 +48,7 @@
 
   private static final String SERIALIZATION_UNIMPLEMENTED = "serialization unimplemented";
 
+  protected String userName;
```

Comment:
Does every physical plan have to have a userName?
for OOP, I do not think its a good design.

Reply:
Yes, I will change it to AuthorPlan only. Thank you for your comment!

Reply:
I have a different opinion. Every physical plan is issued by a user and keeping this information will help us tracking user behaviors in the future.